### PR TITLE
Improve list item UI for directory

### PR DESCRIPTION
**Pull request by shank1xt**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

feature

* **What is the current behavior?** (You can also link to an open issue here)

ui for list item and card is not complete

* **What is the new behavior (if this is a feature change)?**

improves ui for list item and card

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

no

* **Other information**:

### CODE REVIEW

File: packages/apps/src/directory/ListingCard.tsx [hunk 1]
```diff
@@ -66,8 +79,11 @@ const ListingCard: React.FC<ListingCardProps> = (props) => {
         </Link>
       )}
 
-      <CardContent disableGutterBottom sx={{ height: '100%' }}>
-        <Stack alignItems="center" direction="row" gap={2}>
+      <CardContent
+        disableGutterBottom
+        sx={{ height: '100%', mt: is_listing_brand_image_enabled ? -6 : 0 }}
```
Comment (clodal) [inline packages/apps/src/directory/ListingCard.tsx]: Odd to have -6 here

File: packages/apps/src/directory/DirectoryTemplate.tsx [hunk 2]
```diff
@@ -69,11 +69,19 @@ const DirectoryTemplate: React.FC<DirectoryTemplateProps> = (props) => {
...
 
+  const isListMode = variant === PaginatedQueryViewVariantEnum.List
+
   // State: Bottom drawer
   const directoryListingsJsx = (
     <PaginatedListings
-      gridItemProps={gridItemProps}
-      gridProps={gridProps}
+      gridItemProps={
+        isListMode
```
Comment (clodal) [inline packages/apps/src/directory/DirectoryTemplate.tsx]: Are we able to manage these scenarios in the switch case instead? Want to keep the view clean

**Review by clodal — CHANGES_REQUESTED** [review]

**Comment by clodal** [comment]
@shank1xt Able to share screenshots as to what we're achieving here? Thanks

**Comment by shank1xt** [comment]
Need to add the hero images for the brands, but here we are so far.

image

image

**Review by clodal — APPROVED** [review]

**Pull request merged**